### DISCUSSION TITLE
Using existing storage account

### DISCUSSION THREAD

**Discussion by simonarbel** [microsoft/finops-toolkit#825] [General]
Hello

Customer has existing FOCUS exports in a storage blob.  
Can I deploy the Finops hub and direct it to the existing exports?

**Comment by flanakin** [comment]
Not without a bunch of manual setup to reconfigure what storage account to use and then manually run pipelines on the existing data. It would be simpler to just configure a new export.

One thing I haven't tried however, you might be able to configure the existing storage account as a remote hub. I'm not 100% sure about this and would need @MSBrett to chime in, but it may be worth a shot.

**Comment by arthurclares** [comment]
Soryr, I missed to update this thread. Simon, Brett and I met to discuss this.
Very high level steps are: In data factory, change linked service URL,  then assign permissions for Azure Data Factory on Storage Account, and create config folder.
Simon will submit a PR to document these steps once he is done.
